system/jlib/jstream.cpp

```diff
@@ -530,11 +530,10 @@ extern jlib_decl std::pair<const char *, const char *> peekKeyValuePair(IBuffere
     }
 }
 
-extern jlib_decl bool peekStringList(std::vector<const char *> matches, IBufferedSerialInputStream & in, size32_t & len)
+const char * peekStringList(std::vector<size32_t> & matchOffsets, IBufferedSerialInputStream & in, size32_t & len)
```

Comment:
This should have previously been a reference parameter - but the unit test didn't pick it up because it didn't check the lengths matched.  Now fixed.